src/widgets/communication/ui/communication.module.scss

```diff
@@ -1,39 +1,38 @@
 .communication-content {
   display: flex;
   flex-direction: column;
-  gap: 26px;
-}
 
-.communication-wrapper {
-  display: flex;
-  flex-direction: row;
-  gap: 100px;
-  align-items: center;
-  justify-content: flex-start;
+  .communication-wrapper {
+    display: flex;
+    flex-direction: row;
+    gap: 100px;
+    align-items: center;
 
-  @include media-tablet-large {
-    flex-direction: column;
-    gap: 40px;
-    align-items: flex-start;
+    @include media-tablet-large {
+      flex-direction: column;
+      gap: 40px;
+      align-items: flex-start;
+    }
   }
-}
 
-.communication-paragraph {
-  padding-top: 5px;
-}
+  .communication-subtitle,
```

Comment:
Why do we need to overwrite styles for subtitle?

Reply:
Previously padding-top was set for the paragraph but it's better to use padding-bottom. I changet it to padding-bottom for the subtitle and one paragraph.
With the default styles it looks worse in my opinion.
![image](https://github.com/user-attachments/assets/a0c31348-9ee9-49be-8541-361d073581eb)

Reply:
We discussed it with Anna and decided to leave it as is